style.css

```diff
@@ -1,531 +1,641 @@
+    right: 0;
+    width: 4.25rem;
+    height: 10.94rem;
+    background-image: url('./images/Frame\ \(1\).svg');
+    background-size: cover;
+    background-position: right bottom;
+  }
 
-.info3 {
-  font-weight: 700;
-  font-size: 40px;
-  display: flex;
-  align-items: center;
-  text-align: center;
-  color: #172b4d;
-  padding-bottom: 8%;
-}
+  .right-background2 {
+    display: none;
+  }
 
-.name-email {
-  height: 48px;
-  width: 88%;
-  margin-left: 6%;
-  border: 1px solid #d0d9d4;
-  padding-left: 10px;
-  border-radius: 4px;
-  margin-top: 2%;
-  margin-bottom: 5%;
-  font-style: italic;
-  color: #6b778c !important;
-}
+  footer p {
+    font-style: normal;
+    font-weight: 400;
+    font-size: 20px;
+    line-height: 32px;
+    text-align: center;
+    color: #42526e;
+  }
 
-textarea {
-  color: #42526e;
-  padding-top: 2%;
-  padding-left: 4%;
-  padding-right: 8%;
-  padding-bottom: 40%;
-  width: 88%;
-  font-family: 'Inter', sans-serif;
-  font-style: normal;
-  font-weight: 400;
-  font-size: 15px;
-  line-height: 24px;
-  border: 1px solid #60c095;
-  margin-top: 2%;
-  margin-bottom: 4%;
-  margin-left: 6%;
-  border-radius: 4px;
-}
+  .footer-line {
+    display: none;
+  }
 
-.my-resume2 {
-  display: flex;
-  align-items: center;
-  color: #fff;
-  border: #fff;
-  gap: 10px;
-  width: 35%;
-  height: 48px;
-  bottom: 8px;
-  padding-left: 4%;
-  margin-left: 110px;
-  background: #36b37f;
-  letter-spacing: 1px;
-  border-radius: 4px;
+  footer .social-icons {
+    margin-bottom: 20px;
+  }
 }
 
-.footer .info4 {
-  font-weight: 400;
-  font-size: 16px;
-  display: flex;
-  position: absolute;
-  justify-content: center;
-  align-items: center;
-  text-align: center;
-  margin-left: 21%;
-  line-height: 1.5;
-  color: #42526e;
-  padding-top: 10%;
+@media (min-width: 600px) {
+  .background-bottom-left {
+    top: 10rem;
+  }
 }
 
-.footer .last-img3 {
-  width: 100%;
-  height: auto;
-  margin-left: 82%;
-  margin-top: -16%;
+@media (min-width: 768px) {
+  ul {
+    list-style: none;
+  }
+
+  .desktop-menu {
+    display: block;
+    position: relative;
+  }
+
+  .desktop-menu ul {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    gap: 25px;
+    position: absolute;
+    top: 0;
+    left: 50%;
+    transform: translate(-50%, 50%);
+  }
+
+  .desktop-menu ul li a {
+    text-decoration: none;
+    font-family: 'Inter', sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 20px;
+    line-height: 32px;
+    text-align: center;
+    color: #050e1d;
+  }
+
+  .desktop-menu li a:hover {
+    color: #008552;
+  }
+
+  .desktop-menu li a:active {
+    color: #36b37f;
+  }
+
+  .mobile-menu-icon {
+    display: none;
+  }
+
+  .home {
+    background: url(./images/Header-llustration-desktop.svg) no-repeat;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    background-size: cover;
+    padding-top: 200px;
+  }
```

Comment:
[Required] Good job on your progress so far :clap:, However, as the previous reviewer mentioned, your web page is not entirely responsive to all screen sizes. This means that from 768 to 940, your `background image`  should be responsive exactly as shown in the Figma Design. Please check below for clarity. having a responsive design ensures a consistent and user-friendly experience across devices, expands your reach to mobile users, improves SEO, saves costs and time, and prepares you for future technological advancements. It is an essential aspect of modern web and application development. :slightly_smiling_face: 

The below table shows the difference between yours and the Figma Design, Please pay close attention to the red circle in your design.
<table>
    <tr>
      <td>Your Design </td>
      <td>Figma Design</td>
    </tr>
    <tr>
      <td>

![Reviewer Required](https://github.com/newtonah0/Mobile-Portfolio/assets/43528594/aeeb2077-e4d7-4893-b7ba-49146cc93200)

</td>
      <td>

![Figma Design For required](https://github.com/newtonah0/Mobile-Portfolio/assets/43528594/393063f9-57c3-462c-86f5-f58cfdb1a5ae)

</td>
    </tr>
  </table>

Reply:
so what do you advice me to solve this?